src/psyclone/tests/psyir/transformations/extract_test.py

```diff
@@ -429,114 +379,92 @@ def test_node_list_dynamo0p3():
-    schedule, _ = otrans.apply(schedule.children[1])
-    # Now enclose the parallel region within an ExtractNode (inserted
-    # at the previous location of the OMPParallelDoDirective
-    schedule, _ = etrans.apply(schedule.children[1])
-
+    schedule, _ = etrans.apply(schedule.children[0:3])
     code = str(psy.gen)
-    output = (
-        "      ! ExtractStart\n"
-        "      ! CALL write_extract_arguments(argument_list)\n"
-        "      !\n"
-        "      !$omp parallel do default(shared), private(i,j), "
-        "schedule(static)\n"
-        "      DO j=2,jstop+1\n"
-        "        DO i=2,istop\n"
-        "          CALL compute_cv_code(i, j, cv_fld%data, "
-        "p_fld%data, v_fld%data)\n"
-        "        END DO\n"
-        "      END DO\n"
-        "      !$omp end parallel do\n"
-        "      !\n"
-        "      ! ExtractEnd\n")
-    assert output in code
-
-
-def test_node_list_ompparallel_gocean1p0():
-    ''' Test that applying Extract Transformation on a list of Nodes
-    enclosed within an OMP Parallel Region produces the correct result
-    in GOcean1.0 API. '''
-    from psyclone.transformations import GOceanOMPLoopTrans, OMPParallelTrans
 
-    etrans = GOceanExtractTrans()
-    ltrans = GOceanOMPLoopTrans()
-    otrans = OMPParallelTrans()
-
-    # Test a Loop nested within the OMP Parallel DO Directive
-    psy, invoke = get_invoke("single_invoke_three_kernels.f90",
-                             GOCEAN_API, idx=0, dist_mem=False)
-    schedule = invoke.schedule
-
-    # Apply GOceanOMPParallelLoopTrans to the first two Loops
-    schedule, _ = ltrans.apply(schedule.children[0])
-    schedule, _ = ltrans.apply(schedule.children[1])
-    # and enclose them within a parallel region
-    schedule, _ = otrans.apply(schedule.children[0:2])
-    # Now enclose the parallel region within an ExtractNode (inserted
-    # at the previous location of the OMPParallelDirective
-    schedule, _ = etrans.apply(schedule.children[0])
-
-    code = str(psy.gen)
-    output = (
-        "      ! ExtractStart\n"
-        "      ! CALL write_extract_arguments(argument_list)\n"
-        "      !\n"
-        "      !$omp parallel default(shared), private(i,j)\n"
-        "      !$omp do schedule(static)\n"
-        "      DO j=2,jstop\n"
-        "        DO i=2,istop+1\n"
-        "          CALL compute_cu_code(i, j, cu_fld%data, "
-        "p_fld%data, u_fld%data)\n"
-        "        END DO\n"
-        "      END DO\n"
-        "      !$omp end do\n"
-        "      !$omp do schedule(static)\n"
-        "      DO j=2,jstop+1\n"
-        "        DO i=2,istop\n"
-        "          CALL compute_cv_code(i, j, cv_fld%data, "
-        "p_fld%data, v_fld%data)\n"
-        "        END DO\n"
-        "      END DO\n"
-        "      !$omp end do\n"
-        "      !$omp end parallel\n"
-        "      !\n"
-        "      ! ExtractEnd\n")
+    # Node that this code was manually created (see #646), and might need
```

Comment:
s/Node/Note/

Reply:
With not so much as a nod I noted that it's the wrong node.